src/hera/workflows/runner.py

```diff
@@ -122,7 +123,53 @@ def _map_keys(function: Callable, kwargs: dict) -> dict:
     return mapped_kwargs
 
 
-def _runner(entrypoint: str, kwargs_list: Any) -> str:
+def _save_outputs(function_results, output_destinations):
```

Comment:
Any chance this can be documented a bit more? Reading it is relatively straight forward, but some documentation, and types, would be very useful for readability

Reply:
Sure thing!